static/css/style.css

```diff
@@ -28,6 +28,9 @@ body, html {
   background-color: #0099ff !important;
 }
 
+.card-header:first-child {
+    border-radius: 0.8em 0.8em 0 0;
```

Comment:
is that border-radius is used all over? or just your personal style opinion? It's more important to be consistent than just opiniated guesses

Reply:
yes is no in line with the boarder of the card

Reply:
Did a new commit in this PR